common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -94,7 +94,7 @@ export type ICommittedProposal = {
     commitSequenceNumber: number;
 } & IApprovedProposal;
 
-// @internal
+// @alpha
 export interface IConnect {
```

Comment:
I think we're going to need Loop to use the ./internal subpath for this one to avoid another server release.

Reply:
upgrading from internal to alpha should be doable in a patch, as it only increases exposure.

Reply:
That's reasonable.  There's a single encapsulated usage.